src/coreclr/tools/ILVerify/README.md

```diff
@@ -40,11 +40,41 @@ The code is split into three projects:
 
 ## Tests
 
-To test the ILVerification library we have small methods checked in as .il files testing specific verification scenarios. These tests live under [src/ILVerification/tests/ILTests](../ILVerification/tests/ILTests). Tests are grouped into .il files based on functionalities they test. There is no strict policy here, the goal is to have a few dozen .il files instead of thousands containing each only a single method.
+To test the ILVerification library we have small methods checked in as .il files testing specific verification scenarios. These tests live under [src/tests/ilverify/ILTests](../../../tests/ilverify/ILTests). Tests are grouped into .il files based on functionalities they test. There is no strict policy here, the goal is to have a few dozen .il files instead of thousands containing each only a single method.
 
 The test project itself is under [src/tests/ilverify](../../../tests/ilverify)
 
- Method names in the .il files must follow the following naming convention:
+### Building and Running Tests
+
+As the test project is marked with priority=1, simply building the test projects from the root of the project is not enough. For the initial build of priority=1 in release mode one must run the following:
```

Comment:
```suggestion
As the test project is marked with priority=1, simply building the test projects from the root of the project is not enough. For the initial build of priority=1 in release mode run the following:
```
Nit: We typically use more direct doc style in this repo. I am not commenting on all of these.

Reply:
I see, I was not aware! I've adjusted the documentation style, I hope its okay now!